src/api_server.cc

```diff
@@ -204,6 +210,13 @@ void SendMetadataRequest(std::vector<json::value>&& entries,
   request << boost::network::header("Authorization", auth_header);
   request << boost::network::body(request_body);
   http::client::response response = client.post(request);
+  if (status(response) != 200) {
+    throw boost::system::system_error(
+        boost::system::errc::make_error_code(boost::system::errc::not_connected),
```

Comment:
I don't believe not_connected is the best exception to throw here, but if it's just a matter of getting "something" to throw, it's fine.

Reply:
Yes, that code is never examined.